.github/workflows/deploy-prod.yml

```
@@ -1,14 +1,13 @@
-name: Deploy to production environment
+name: Deploy to staging environment
```

You have a typo here, it should be `production`
```suggestion
name: Deploy to production environment
```